openshiftcontrolplane/v1/types.go

```diff
@@ -189,6 +189,11 @@ type OpenShiftControllerManagerConfig struct {
 	Ingress            IngressControllerConfig          `json:"ingress"`
 	ImageImport        ImageImportControllerConfig      `json:"imageImport"`
 	SecurityAllocator  SecurityAllocator                `json:"securityAllocator"`
+
+	// featureGates captures the current set of OpenShift feature gates that are of interest to
+	// components of the OpenShift Controller Manager.  NOTE: this field will possibly be deprecated
+	// in a future OpenShift release if the feature gate mechanism in OpenShift is removed.
+	FeatureGates map[string][]string `json:"featureGates"`
```

Comment:
My testing of things on Friday as I recall came to the conclusion that anything set to false did not get included in what ultimately bubbled up.  i.e. it got filtered at the library-go level.

I'll double check if we move forward down this path.  But if I'm incorrect, then yes, ocm-o/ocm would need to account for it.

In the interest of code reuse and expediency, I at least am OK with it.

Reply:
I'll pull your suggestion in manually @adambkaplan since I'll need to run `make update` anyway to get the verifications to pass ... save on some CI churn

thanks

Reply:
godoc update pushed @adambkaplan